cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -594,6 +594,14 @@ func TestPolicyUpdate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		"Error on mount apply failing": {
+			initState: "localhost-uptodate",
+			readOnlyDirs: []string{
+				// In order to make the mount manager fail, we need create a not empty mounts directory inside the user folder.
+				fmt.Sprintf("run/users/%s/mounts", currentUID),
```

Comment:
It might be a little weird to rely on readOnlyDirs to create the directory that we need for the failure.
I can add another field in the test case struct to handle it, if you think it's better.

Reply:
Well, not really. We already saw that, when writing in ext4 'safe mode', Go does not care if we replace a file with readOnly permissions and that the only way  to make the mount manager fail (when it comes to permissions) is to have a directory called mounts, rather than a file with readOnly permissions. So if we initial state already have a mounts file, we can't make the mount manager fail (at least for now), unless we remove the file and replace it with a non empty directory.

Reply:
So, the comment is not correct, it’s "we need to change the file by a non empty RO directory". Does it really need to be non empty btw?

I would still like if we can minimize the amount of parameters, if possible at all. However, the new logic may change the other failure, no?